src/examples/tutorial/ascent_intro/cpp/plot_binning_2d.py

```diff
@@ -0,0 +1,49 @@
+import yaml #pip install --user pyyaml
+import pandas as pd
```

Comment:
we can drop pandas here, and I will install pyyaml as part of the extra container setup.